test/integration/main.cpp

```diff
@@ -901,4 +903,44 @@ TEST_F(ZNCTest, ModuleCrypt) {
     Z;
 }
 
+TEST(UnixSocketTest, Connect) {
+    // Create a config
+    QTemporaryDir tempDir;
+    QDir dir(tempDir.path());
+    QString socketName = dir.filePath("socket");
+    dir.mkdir("configs");
+    QFile config(dir.filePath("configs/znc.conf"));
+    ASSERT_TRUE(config.open(QIODevice::WriteOnly | QIODevice::Text));
+    Z;
+
+    QTextStream out(&config);
+    // FIXME: Hardcoding a version is bad
```

Comment:
Is version necessary there?
If it is, to avoid hardcoding it, you can also use --makeconf, and append stuff to config, like in ModuleCrypt test above.